src/utils/geo.js

```diff
@@ -0,0 +1,202 @@
+import Cookies from 'js-cookie';
+import Random from './random';
+
+const earth = 'XX',
+	negativePrefix = 'non-',
+	samplingChar = '/';
+
+let geoData = null,
+	cache = [];
+
+function isCountryWithSampling(geo) {
+	return value => !value.startsWith(negativePrefix) && value.includes(geo + samplingChar);
```

Comment:
What about `non-US/50`? Are we going to support it?

Reply:
What about IE11 (includes)? 😈

Reply:
Nope `non-US/50` isn't supported